src/plotman/archive.py

```diff
@@ -9,37 +9,65 @@
 import psutil
 import texttable as tt
 
-from plotman import job, manager, plot_util
+from plotman import configuration, job, manager, plot_util
 
 # TODO : write-protect and delete-protect archived plots
 
-def spawn_archive_process(dir_cfg, all_jobs):
+def spawn_archive_process(dir_cfg, arch_cfg, all_jobs):
     '''Spawns a new archive process using the command created
     in the archive() function. Returns archiving status and a log message to print.'''
 
-    log_message = None
+    log_messages = []
     archiving_status = None
 
     # Look for running archive jobs.  Be robust to finding more than one
     # even though the scheduler should only run one at a time.
-    arch_jobs = get_running_archive_jobs(dir_cfg.archive)
+    arch_jobs = get_running_archive_jobs(arch_cfg)
 
     if not arch_jobs:
-        (should_start, status_or_cmd) = archive(dir_cfg, all_jobs)
+        (should_start, status_or_cmd, archive_log_messages) = archive(dir_cfg, arch_cfg, all_jobs)
+        log_messages.extend(archive_log_messages)
         if not should_start:
             archiving_status = status_or_cmd
         else:
-            cmd = status_or_cmd
-            # TODO: do something useful with output instead of DEVNULL
-            p = subprocess.Popen(cmd,
+            args = status_or_cmd
+
+            log_file_path = dir_cfg.create_log_path(group='transfer', time=pendulum.now())
+
+            # TODO: CAMPid 09840103109429840981397487498131
+            try:
+                open_log_file = open(log_file_path, 'x')
+            except FileExistsError:
+                log_messages.append(
+                    f'Archiving log file already exists, skipping attempt to start a'
+                    f' new plot: {log_file_path!r}'
+                )
+                return (False, log_messages)
+            except FileNotFoundError as e:
+                message = (
+                    f'Unable to open log file.  Verify that the directory exists'
+                    f' and has proper write permissions: {log_file_path!r}'
+                )
+                raise Exception(message) from e
+
+            # Preferably, do not add any code between the try block above
+            # and the with block below.  IOW, this space intentionally left
+            # blank...  As is, this provides a good chance that our handle
+            # of the log file will get closed explicitly while still
+            # allowing handling of just the log file opening error.
+
+            with open_log_file:
+                # start_new_sessions to make the job independent of this controlling tty.
+                p = subprocess.Popen(**args,
                     shell=True,
-                    stdout=subprocess.DEVNULL,
+                    stdout=open_log_file,
                     stderr=subprocess.STDOUT,
                     start_new_session=True)
-            log_message = 'Starting archive: ' + cmd
+            log_messages.append('Starting archive: ' + args['args'])
```

Comment:
Include the log path: 

```python
log_messages.append('Starting archive: ' + args['args'] + f". Saving logs to {log_file_path}")
```

Reply:
I think this matches the plot starting.  But yeah, this is a great way to make the log line more useful, thanks.

```suggestion
            log_messages.append(f'Starting archive: {args["args"]} ; logging to {log_file_path}')
```